plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -48,7 +49,12 @@ const SearchResultComponent = ({ children }: Props) => {
     return <EmptyState missing="data" title="Sorry, no results were found" />;
   }
 
-  return children({ results: value.results });
+  return (
+    <>
+      {children({ results: value.results })}
+      <SearchResultPager />
```

Comment:
Just wondering, is the techdocs in context search even using the `SearchResult` component, if not I'm not sure about this 😆 

But I will have a look. It will probably mean that I have to write about it in the changelog as users have to add it in their app instead.

Reply:
👍 Done